src/libponyc/CMakeLists.txt

```diff
@@ -121,9 +121,22 @@ endif (NOT MSVC)
 # build a standalone version of libponyc.a with all needed dependencies linked statically
 if (MSVC)
     # TODO
-    #file(GLOB_RECURSE LLVM_OBJS "${PROJECT_SOURCE_DIR}/../../build/build_libs/llvm/src/llvm/Release/lib/*.lib")
 elseif(${CMAKE_HOST_SYSTEM_NAME} MATCHES "Darwin")
-    # TODO
+    # on macos we dont have no static libc++
+    # so when we want to use this lib
+    # we need to additionally link libc++ and libz (for llvm)
+    file(GLOB_RECURSE LLVM_OBJS "${PROJECT_SOURCE_DIR}/../../build/build_libs/llvm/src/llvm//lib/libLLVM*.a")
```

Comment:
I can't remember why, but the GLOB_RECURSE had problems on Linux once I started using.

Reply:
It is finding all the libs and the resulting static library has all we need. And this is scoped to Darwin only, so, unless this becomes a problem on M1 mac, i think this is fine, but thanks for the heads-up.

Reply:
How was this tested on MacOS?

Reply:
It was tested by taking a pony program and linking it to the resulting `libponyc-standalone.a`.

Reply:
what program? does the program fully exercise the API or could there be errors that get hidden by dead code elimination?